### ISSUE TITLE
Suggestion to make default value of chunk_size dependent on language

### ISSUE THREAD

**Issue by oep42** [m-bain/whisperX#515]
The [recently added](https://github.com/m-bain/whisperX/pull/445) argument `--chunk_size` has a default value of 30. It may be desirable to have different default values for different languages. It seems that the current default chunk duration (segment duration) of 30 is too long for most languages.

I suggest:

1. Determine the most appropriate default value of chunk_size for each of the supported languages.
2. Based on these most appropriate default values:  Make the default value of `--chunk_size` dependent on the value of `--language`.

When this has been implemented, the `--chunk_size` argument only has to be used when you want to overrule the default value that is the most appropriate for the current language.

**Comment by m-bain** [comment]
yes less context will result in worse transcription accuracy, and lower throughput (longer time to finish transcribing file).

I think for some usecases people don't care about amount of text in chunk, i will leave this up to user to decide (its a trade-off). But definitely its worth detailing this more in the README.

**Comment by cleverestx** [comment]
In your experience guys, what should the chink size me set for Japanese audio videos? No long speeches in them, just occasional phrases and short sentences from time to time.
